### PR TITLE
Add static location for pics, change all linked pics in Firefly Wallet.md

### PR DESCRIPTION
**Pull request by Phyloiota**
# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix
- Documentation Enhancment

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

**Comment by Phyloiota** [comment]
Added all pics into the static folders, changed all the links in "Firefly Wallet" to the new static links in /src/IMG/learn/firefly
Closing the other Firefly PR
All updated content is in this one

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ As a result, you can attach new messages to only one place: a block at the end o
 
 **Blockchain bottleneck**
 
-![blockchain-bottleneck](https://user-images.githubusercontent.com/77154511/127757399-8feb30cc-c5c2-4c9f-abf0-7d4584fa2b6c.gif)
+![blockchain-bottleneck](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/blockchain-bottleneck.gif)
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:28]:
```suggestion
![blockchain-bottleneck](img/learn/blockchain-bottleneck.gif)
```

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Rather than being limited to a single place for attaching new messages, you can
 
 **Tangle bottleneck**
 
-![tangle-bottleneck](https://user-images.githubusercontent.com/77154511/127757420-7c85a003-57e0-49f7-a534-defe96e10df6.gif)
+![tangle-bottleneck](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/tangle-bottleneck.gif)
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:40]:
```suggestion
![tangle-bottleneck](img/learn/tangle-bottleneck.gif)
```

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 3]
```diff
@@ -69,4 +69,4 @@ To allow them to recognize milestones, all nodes in the same IOTA network are co
 
 To make sure that new messages always have a chance of being confirmed, the Coordinator sends indexed milestones at regular intervals. This way, nodes can compare the indexes of their milestones to check whether they are synchronized with the rest of the network.
 
-![coordinator](https://user-images.githubusercontent.com/77154511/127796118-08915c1d-438d-4d9a-8432-893c6c5b4f0d.gif)
+![coordinator](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/milestones.gif)
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:72]:
```suggestion
![coordinator](img/learn/milestones.gif)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 4]
```diff
@@ -84,16 +87,16 @@ If different Users aim to use the same Firefly Wallet App, or if you have IOTA T
...
+- **Securely store your Stronghold Backup File and the Stronghold Password!**
+-  **Store it in multiple independent locations - digital and physical!**
 
 
 
 
 ### Firefly Dashboard
 
-![image](https://user-images.githubusercontent.com/77154511/126114270-96c27168-6b49-4a57-95e4-c1ec1b057249.png)
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/dashboard1.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:99]:
```suggestion
![image](img/learn/firefly/dashboard1.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 5]
```diff
@@ -109,7 +112,7 @@ Important to mention here: Users can only set up further Wallets if the first Wa
 
 ### Wallet Detail view
 
-![image](https://user-images.githubusercontent.com/77154511/126114427-df51d0e3-a44e-48e0-a4c6-f4050dcff379.png)
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/wallet%20detail%20view1.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:115]:
```suggestion
![image](img/learn/firefly/wallet%20detail%20view1.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 6]
```diff
@@ -121,60 +124,60 @@ Also, you have will find **3 little dots** next to the Wallet balance.
 
 **Clicking on those dots opens a popup menu.**
 
-- <img src="https://user-images.githubusercontent.com/77154511/126114811-d792208d-8d76-47d8-b1c3-0cff5cdc6a54.png" width="400" height="600"/>
+<img src="https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/account%20detail%20customize1.png" width="400" height="600"/>
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:127]:
```suggestion
<img src="img/learn/firefly/account%20detail%20customize1.png" width="400" height="600"/>
```

File: docs/learn/wallets/firefly-wallet.md [hunk 7]
```diff
@@ -121,60 +124,60 @@ Also, you have will find **3 little dots** next to the Wallet balance.
...
-- [ ] **Hide Wallet**
+- **Hide Wallet**
 
 - Empty Wallets can be hidden to keep your Main Wallet overview organized. If a wallet still contains funds you will not be able to hide it by clicking this option but will be offered to transfer the funds to another Wallet and hide the Wallet afterward. A hidden Wallet can be shown again by enabling the function "show hidden wallets" in the advanced account settings.
 
 
 #### Transaction detail view.
 
-![image](https://user-images.githubusercontent.com/77154511/126114552-383a3f34-e665-46a7-a3d8-4e6277811ce3.png)
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/transaction%20details%20view1.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:146]:
```suggestion
![image](img/learn/firefly/transaction%20details%20view1.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 8]
```diff
@@ -184,9 +187,9 @@ The Addresses belonging to you are highlighted with the name of your wallet in b
 
 - The Send and Receive functions are available in the main Wallets dashboard, and also in every Wallet detail view.
 
-- [ ] **Receive**
+- **Receive**
 
-- <img src="https://user-images.githubusercontent.com/77154511/126114880-0ffeb841-22ef-4858-a096-47a3b4ab333d.png" width="400" height="600"/>
+<img src="https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/receive%20funds1.png" width="400" height="600"/>
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:192]:
```suggestion
<img src="img/learn/firefly/receive%20funds1.png" width="400" height="600"/>
```

File: docs/learn/wallets/firefly-wallet.md [hunk 9]
```diff
@@ -195,9 +198,9 @@ The receive address of the selected Wallet is displayed as a QR Code and as a wr
 - If a user has received, but not yet spent funds from that receiving address, the user can manually decide to generate a fresh receive address using the small "refresh" button above the QR Code.
 This function is an essential privacy feature that is a standard approach in Crypto to reduce the traceability of your balances
 
-- [ ] **Send**
+- **Send**
 
-- <img src="https://user-images.githubusercontent.com/77154511/126115936-253f9e94-2cef-460f-9314-020b6843730d.png" width="400" height="600"/>
+<img src="https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/send%20funds1.png" width="400" height="600"/>
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:203]:
```suggestion
<img src="img/learn/firefly/send%20funds1.png" width="400" height="600"/>
```

File: docs/learn/wallets/firefly-wallet.md [hunk 10]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+### How to set up a Ledger Nano with Firefly
+
+Note: If you are an existing IOTA user and secured IOTA with a Ledger before the Chrysalis network upgrade on 28 April 2021, you will need to migrate your tokens over to the new network before you can begin using Firefly. There is a comprehensive written guide and a video guide to help you through the migration process.
+
+### Creating a new Ledger profile
+
+Creating a new Ledger profile in Firefly is very quick and simple. Once you have downloaded Firefly, open the app and follow the setup instructions. You first need to review and accept the terms and conditions, set a theme (dark or light), and choose a profile name. Select "Create a new wallet".
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/setup_a_wallet1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:255]:
```suggestion
![image](img/learn/firefly/setup_a_wallet1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 11]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+Press "I want a hardware wallet" to begin the Ledger profile setup process.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/create_a_wallet1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:259]:
```suggestion
![image](img/learn/firefly/create_a_wallet1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 12]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+On the next page, you will be asked to set a PIN code. This PIN is used to log in to your wallet and blocks other people from logging in and viewing your balance transaction history. Choose a PIN that only you know. Re-enter your PIN on the next page to confirm.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/set_pin1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:263]:
```suggestion
![image](img/learn/firefly/set_pin1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 13]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+Now, Firefly will check that your Ledger Nano is connected. Make sure the IOTA app is open on your device and that Ledger Live is closed. If you have connection problems follow the "Tips if your Ledger isn't connecting" guide in Firefly.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/connect_ledger1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:267]:
```suggestion
![image](img/learn/firefly/connect_ledger1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 14]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+Hit "Continue", and that's all there is to it. It really is that simple. You can now enter the dashboard and begin exploring Firefly.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/setup_complete1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:271]:
```suggestion
![image](img/learn/firefly/setup_complete1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 15]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+### Receiving tokens
+
+If you would like to receive funds to your new Ledger profile you can press the "Receive" button on the dashboard.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/generate_address1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:277]:
```suggestion
![image](img/learn/firefly/generate_address1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 16]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+On this page you can select "Generate address" and Firefly will prompt you to confirm the address matches the one displayed on your Ledger device.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/confirm_receive_address1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:281]:
```suggestion
![image](img/learn/firefly/confirm_receive_address1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 17]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+Scroll through the address on your Ledger device. Confirm it matches by pressing both buttons when your Ledger reads "Ok".
+
+Note: This is a security check to ensure that you are connected to the official, secure version of Firefly. It ensures that no malicious software can alter the address you receive to and that it matches the one generated by your Ledger.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/display_ledger_address1.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:287]:
```suggestion
![image](img/learn/firefly/display_ledger_address1.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 18]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/okay_ledger_display1.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:289]:
```suggestion
![image](img/learn/firefly/okay_ledger_display1.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 19]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+You can now copy the address and share it with a friend or exchange from which you wish to receive tokens. Always make sure the address is the same as the one displayed on your Ledger device.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/wallet1_view_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:293]:
```suggestion
![image](img/learn/firefly/wallet1_view_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 20]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+### Sending tokens
+
+Once you have received tokens to an address, your balance will update and you will be able to view the transaction in the transaction history. To send these tokens, press the "Send" button from the dashboard. Here you can fill in the address you want to send to, enter an amount, and hit "Send".
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/send_payment1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:299]:
```suggestion
![image](img/learn/firefly/send_payment1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 21]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+You will then need to confirm that the full transaction details match what is displayed on your Ledger device. In some cases, where you are not sending the total balance on your address, you will also need to approve a "Remainder address". Scroll through the transaction details on your Ledger. Confirm the details match by pressing both buttons when your Ledger reads "Accept". If they do not match press "Deny".
+
+Note: As with generating addresses, this is a security check to ensure that you are connected to the official, secure version of Firefly. It ensures that no malicious software can alter the contents of your transaction and ensures you are sending the right amount to the correct address.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/confirm_remainder_address1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:305]:
```suggestion
![image](img/learn/firefly/confirm_remainder_address1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 22]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/confirm_transaction1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:307]:
```suggestion
![image](img/learn/firefly/confirm_transaction1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 23]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/dispaly_amount1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:309]:
```suggestion
![image](img/learn/firefly/dispaly_amount1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 24]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/display_accept1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:311]:
```suggestion
![image](img/learn/firefly/display_accept1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 25]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+### Creating wallets and sending internal transfers
+
+Firefly makes it possible to organize your tokens into different wallets within your profile. These tokens are kept separate from one another. Technically speaking, they are stored on addresses generated on different sub-accounts on your Ledger. Press "Create" from the dashboard and choose a name to add another wallet.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/multiple_wallet_view1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:317]:
```suggestion
![image](img/learn/firefly/multiple_wallet_view1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 26]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+
+With multiple wallets, you can then send between them and split your tokens up to organize them however you like. You can select "Internal Transfer" from the Send view and select which wallet you would like to send to.
+
+Note: Your most recently created wallet must receive funds (i.e. not be empty) before you can create a new one. This ensures that your wallets can be more smoothly recovered if you need to restore your Ledger profile on a new device.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/choose_send_from_wallet1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:324]:
```suggestion
![image](img/learn/firefly/choose_send_from_wallet1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 27]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+
+### Restoring an existing Ledger profile
+
+If you need to restore an existing Firefly profile with a new device, or you somehow lose access to Firefly, you can plug in your Ledger and restore an existing profile in a few simple steps. To do so, select "Migrate or restore a wallet" in setup and then "I have a Firefly Ledger backup".
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/setup_a_wallet1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:331]:
```suggestion
![image](img/learn/firefly/setup_a_wallet1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 28]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/create_a_wallet1_ledger.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:334]:
```suggestion
![image](img/learn/firefly/create_a_wallet1_ledger.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 29]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
+Note: Backups of your transaction history for Ledger Nano users are not currently supported but will be added in a later version. Once a profile is restored, Firefly will sync your transaction history. It will always find your balance. In a future version, we will add support for retrieving your full transaction history from a node that stores the complete history of the IOTA Tangle.
+
+### Adding a second Ledger device to Firefly
+
+It is possible to have more than one Ledger device with the same Firefly app. You can add another device by creating an additional profile from the login page. Firefly will always check that the right device is connected and you will receive an error message if you try to generate an address or send a transaction with the wrong device.
+
+![image](https://github.com/iota-community/iota-wiki/blob/add-static-image-locations-phylo/static/img/learn/firefly/ledger_multiple_profile_view1.png)
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:342]:
```suggestion
![image](img/learn/firefly/ledger_multiple_profile_view1.png)
```

File: docs/learn/wallets/firefly-wallet.md [hunk 30]
```diff
@@ -206,16 +209,153 @@ The user now defines the Wallet from where the tokens should be sent in a dropdo
...
+
```
Comment (Dr-Electron) [inline docs/learn/wallets/firefly-wallet.md:343]: Why so many newlines? 😆

**Review by Dr-Electron — APPROVED** [review]

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**